fighthealthinsurance/admin.py

```
@@ -681,8 +684,20 @@ class OngoingChatAdmin(admin.ModelAdmin):
         ),
     )
 
-    @admin.display(description="Messages")
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        if connection.vendor == "postgresql":
+            sql = "COALESCE(jsonb_array_length(chat_history), 0)"
+        else:
+            sql = "COALESCE(json_array_length(chat_history), 0)"
+        return qs.annotate(
+            chat_message_count=RawSQL(sql, [], output_field=IntegerField())
```

Ruff (0.15.7)

[error] Use of `RawSQL` can lead to SQL injection vulnerabilities

(S611)